respy/simulate.py

```diff
@@ -42,12 +42,18 @@ def simulate(params, options):
         options["simulation_seed"],
     )
 
-    simulated_data = simulate_data(state_space, base_draws_sims, optim_paras, options)
+    base_draws_wage = create_base_draws(
```

Comment:
+1 for seed ensures that base_draws_sim and base_draws_wage are different, right?

Reply:
yes. I did not want to add another argument to the options.